libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -58,10 +58,13 @@ $theme: (
 			padding: null
 		),
 		rows: (
-			selected-color: #FFF,
 			disabled-color: #F5F5F5,
 			odd-even-row: odd,
-			odd-even-background: #F0F0F0
+			odd-even-background: #F0F0F0,
+			selected: (
```

Comment:
This change will break all the projects overriding default selected foreground color. It's fine as long as you'll fix all those projects once this is merged because we don't wanna get into situation where some dev updates minor or build version of entry-components just to find out SCSS is not compiling.

Reply:
Acknowledged, I will reverse changes to ensure stability and prevent any potential problems with SCSS compilation, selecting a safer strategy at this moment.